.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+> [!NOTE]
+> In the Title field above, Provide a succinct and descriptive title for the pull request, e.g., "Improve caching mechanism for API calls"
```

Comment:
would you like to reference your [commit guidelines](https://github.com/DigiLive/gitChangelog/wiki/1-Introduction)? It may confuse the requestor a bit.